superset-frontend/src/dashboard/components/dnd/handleScroll/handleScroll.test.ts

```diff
@@ -19,6 +19,7 @@
 import handleScroll from '.';
 
 jest.useFakeTimers();
+jest.spyOn(global, 'clearInterval');
```

Comment:
It appears redundant.

Reply:
Actually it's not. Due to new implementation of fake timer in Jest 27, we have to mock the timer-related function such as `clearInterval`